app/jobs/import_url_job_decorator.rb

```diff
@@ -1,33 +1,39 @@
 # frozen_string_literal: true
 
-# OVERRIDE in v3.5.0 of Hyrax
+# OVERRIDE in v2.9.6 of Hyrax
 module ImportUrlJobDecorator
   # OVERRIDE to gain further insight into the StandardError that was reported but hidden.
-  def copy_remote_file(name)
+  def copy_remote_file(uri, name, headers = {})
```

Comment:
I'm concerned about this method signature change.  I also believe that this probably should be contributed back to Hyrax.  Perhaps the double combo/perhaps main.